### ISSUE TITLE
Clashes with latest Watusi updates

### ISSUE THREAD

**Issue by davidwu503** [Nosskirneh/TextEmojis-Public#6]
Been jailbreaking fine with TextEmojis installed but after I updated to latest Watusi updates, ran out of battery and Jailbroke again ending in respring loop. Downgraded back to second latest versions Watusi and was able to Jailbreak again. Deleted TextEmojis and upgraded to latest versions of Watusi and was able to Jailbreak again. Waiting for your next great updates before I reinstall TextEmojis 👍

**Comment by Nosskirneh** [comment]
I think this behavior is caused by Watusi (as you said it was working perfectly fine before). I’m not sure what I can do about this – debugging that early in the boot process is difficult.

When I had trouble rejailbreaking (I experienced respring loops on one of my spare iOS 11.3.1 devices), I did the following:

1. Jailbreak with tweaks turned off.
2. Open iCleaner and disable all tweaks. No need to respring when it asks you to.
3. Rejailbreak with tweaks turned on (although none loaded) with unc0ver (unc0ver doesn’t need to reboot first as Electra does).
4. Open iCleaner and enable all tweaks. Respring when it asks you too.

It might be a few extra annoying steps, but I found myself to be rebooting or running out of battery on such rare occasions that is was simply worth it. You might try these and see if it works for you. As for actually solving the bug, I’m sadly not sure what I can do about it, sorry.

**Comment by davidwu503** [comment]
Good news. I subsequently found myself in a position where I was stuck on respring loop even with no new tweaks installed. I eventually realised it was lost likely as I had little space left (less than 30GB) so I deleted videos to gain back space and the JB worked. Suspecting maybe it was similiar case with TextEmojis and reinstalled your tweak and forced a JB and it worked.

Your tweak is working great. Well done!

With best regards,

David Wu
Sent from my iPhone

On 1 Aug 2019, at 4:04 PM, Andreas Henriksson <notifications@github.com<mailto:notifications@github.com>> wrote:


I think this behavior is caused by Watusi (as you said it was working perfectly fine before). I’m not sure what I can do about this – debugging that early in the boot process is difficult.

When I had trouble rejailbreaking (I experienced respring loops on one of my spare iOS 11.3.1 devices), I did the following:

  1.  Jailbreak with tweaks turned off.
  2.  Open iCleaner and disable all tweaks. No need to respring when it asks you to.
  3.  Rejailbreak with tweaks turned on (although none loaded) with unc0ver (unc0ver doesn’t need to reboot first as Electra does).
  4.  Open iCleaner and enable all tweaks. Respring when it asks you too.

It might be a few extra annoying steps, but I found myself to be rebooting or running out of battery on such rare occasions that is was simply worth it. You might try these and see if it works for you. As for actually solving the bug, I’m sadly not sure what I can do about it, sorry.
